opentelemetry-sdk/src/trace/span.rs

```diff
@@ -580,6 +580,7 @@ mod tests {
             .expect("span data should not be empty as we already set it before")
             .events;
         let event_vec: Vec<_> = event_queue.iter().take(2).collect();
+        #[allow(clippy::get_first)] // we want to extract first two elements
```

Comment:
nit - Can we can use `.first()` in below line, and get rid of  `#[allow(clippy::get_first)]` attribute.